app/assets/javascripts/modules/form-name.js

```diff
@@ -28,7 +28,7 @@ window.moj.Modules.FormName = {
     var self = this;
 
     if (self.$identifier.val().length > 0 && self.$unknown.is(':checked')) {
-      self.$unknown.attr('checked', false);
```

Comment:
I'll take it out of this one, force push, and then make a new smaller PR for this change.